src/column_store/ColumnStoreHelper.h

```diff
@@ -29,7 +29,7 @@ namespace ColumnStore::Helper {
     template<>
     std::pair<__m512i, __m512i> gather(uint64_t *iSA, uint64_t *columnStart) {
         auto indexRegister = _mm512_set_epi64(iSA[7], iSA[6], iSA[5], iSA[4], iSA[3], iSA[2], iSA[1], iSA[0]);
-        return std::make_pair(_mm512_i64gather_epi64(indexRegister, columnStart, 4), indexRegister);
+        return std::make_pair(_mm512_i64gather_epi64(indexRegister, columnStart, 8), indexRegister);
```

Comment:
@adbexy @vincentadamthefirst I believe this was a copy-paste mistake. Is that correct?

Reply:
This looks correct to me. For 32 bit data we need to scale by factor 4 (4byte = 32bit) and for 64 bit data by 8.